src/docs/product/performance/index.mdx

```diff
@@ -45,3 +45,18 @@ Below the graphs are a list of transactions that may be filtered down based on g
 ## Key Transaction View
 
 If you have transactions you frequently return to; you can mark each one as a key transaction in their corresponding [summary views](/product/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+
+## Trends View
+
+Trends surfaces transactions that have had significant changes in their performance over time. You can find the Trends View by toggling the tab in the Performance homepage's upper right corner.
+
+![trends Homepage](trends-homepage.png)
+
+Trending transactions are calculated first by filtering out transactions with large [throughput](/product/performance/metrics/#throughput-total-tpm-tps) fluctuations, determining the baselines of the two halves of the duration, and calculating their percentage change. Regressions are any percentage changes greater than zero, while improvements are percentage changes less than zero.
+The `Confidence` option allows you to filter trends based on High or Low statistical significance. By default, only high confidence, statistically significant results are shown. Sentry uses a [t-test](https://en.wikipedia.org/wiki/Welch%27s_t-test) to compare the before and after periods of the transaction and assign a confidence score. 
+
+For example, looking at a transaction over two weeks with the default [P50](/product/performance/metrics/#average-transaction-duration) baseline, if the first week had a `p(50)=100s`, and the second a `p(50)=110s`, the percentage is `(110/100)-1 = 0.1`. The example transaction had a performance regression of `10%`.
+
+The more data is available, the more the accuracy of the trend improves. For this reason, trends perform best over long periods and with transactions with large counts.
+
+While we default the baseline calculation to use P50, you can change the baseline calculation via the dropdown. Other options include P75, P95, P99, and average. The baseline determines the percentage change, but you can also use it for the comparison view found in each transaction's dropdown. The comparison view finds the transaction in each period with the duration closest to the baseline and compares their spans.
```

Comment:
`The baseline determines the percentage change, but you can also use it for the comparison view found in each transaction's dropdown.` we're taking out comparison view for GA.